static/src/stylesheets/module/identity/_switchboard.scss

```diff
@@ -133,40 +114,55 @@ $checkbox-size: $gs-gutter / 1.25;
         }
     }
 
-}
+    /*
+    show normal checkboxes on
+    older browsers
+    */
+    @supports (transform: scale(1)) {
```

Comment:
I think all browsers that [support `@supports`](https://caniuse.com/#search=support) also [support `transform: scale()`](https://caniuse.com/#search=scale)